### PR TITLE
[NG] profile page for community->members

### PR DESCRIPTION
**Pull request by vinayaksh42**
**Description**
Profile page 

**[Signed commits](../CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [X] Yes, I signed my commits.

### CODE REVIEW

**Comment by leecalcote** [comment]
@vinayaksh42 this is looking great. I've added this profile template to make adding new members easier:

```
---
name:       // First Last Name
role:       // (e.g. Contributor, Maintainer, MeshMate, Community Manager)
/* image_path: // ./<profile-picture.jpg> */
twitter:    // just the user handle (e.g. Vinayak47427793)
github:     // just the user handle (e.g. vinayaksh42)
status:     // Active or Inactive
linkedin:   // last portion of https://www.linkedin.com/in/<profile-name> (e.g. vinayak-sharma-141096193)
location:   // City, Country
focus:      // [optional] list of Layer5 projects (e.g. Meshery UI, Landscape)
tech:       // [optional] list of skills (e.g. Node.js, React, MongoDB)
joined:     // Date joined in Month, Year (e.g. October, 2020) 
bio:        // One or two paragraphs about the community member. 
---
```

Can you update the following:

- change from "position" to "role"
- add "joined" (which should show on their profile page as "Member since: _date_")
- change from "skills" to "focus" (which will be a list of Layer5 projects that they are focused on)
- change "technologies" to "tech" (shorter)

File: src/components/Profile-Page/index.js [hunk 1]
```diff
@@ -0,0 +1,84 @@
+import React from "react";
+import { Link } from "gatsby";
+import { ProfileCardWrapper } from "./ProfilePage.style";
+import { FaTwitter, FaGithub, FaLinkedin } from "react-icons/fa";
+import MeshMateIcon from "../../svgs/meshmate-icon.svg";
+import MeshMateIconWhite from "../../svgs/meshmate-icon.svg";
+import Image from "../image";
+import {Col, Container, Row} from "../../reusecore/Layout";
+
+const ProfileCard = (props) =>{
```
Comment (leecalcote) [inline src/components/Profile-Page/index.js:10]: @vinayaksh42 how do we avoid repeating this const between the Profile-Page and Profile-Card components?

**Review by leecalcote — APPROVED** [review]

File: src/components/Profile-Page/index.js (hunk 1 above)
Comment (Tanuj22) [reply src/components/Profile-Page/index.js:10]: I think this is a typo on @vinayaksh42 part. If I am not wrong the contents inside this function are different. If so this should be changed to something like `ProfileInformation` and also this should be sections folder

Comment (Jashpatel1) [reply src/components/Profile-Page/index.js:10]: Even, I dont think the usage of const is required. It can be directly accessed.

**Pull request merged**